modules/telco-ran-gitops-operator-and-ztp-plugins.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/ran/telco-ran-ref-du-components.adoc
+
+:_content-type: REFERENCE
+[id="telco-ran-gitops-operator-and-ztp-plugins_{context}"]
+= {gitops-shortname} and {ztp} plugins
+
+New in this release::
+* GA support for inclusion of user custom CRs in Git for {ztp} deployments
+
+* {ztp} independence from deployed cluster version
+
+Description::
+{gitops-shortname} and {ztp} plugins provide a {gitops-shortname} driven infrastructure for managing cluster deployment and configuration.
+Cluster definitions and configurations are maintained as a declarative state in Git.
+ZTP plugins provide support for generating installation CRs from the `SiteConfig` CR and automatic wrapping of configuration CRs in policies based on `PolicyGenTemplate` CRs.
+
+You can deploy and manage multiple versions of {product-title} on managed clusters with the baseline reference configuration CRs in a `/source-crs` subdirectory provided that subdirectory also contains the `kustomization.yaml` file.
+You add custom CRs to this subdirectory that you use with the predefined CRs that are specified in the `PolicyGenTemplate` CRs.
+This allows you to tailor your configurations to suit your specific requirements. This also provides ZTP version independence between managed clusters and the hub cluster.
+
+Limits::
+* 300 `SiteConfig` CRs per ArgoCD application.
+You can use multiple applications to achieve the maximum number of clusters supported by a single hub cluster.
+
+* Content in the `/source-crs` folder in Git overrides content provided in the {ztp} plugin container.
+Git takes precedence in the search path.
+
+* Add the `/source-crs` folder in the same directory as the `kustomization.yaml` file, which includes the `PolicyGenTemplate` as a generator.
++
+[NOTE]
+====
+Alternative locations for the `/source-crs` directory are not supported in this context.
+====
+
+Engineering considerations::
+* To avoid confusion or unintentional overwriting of files when updating content, use unique and distinguishable names for custom CRs in the `/source-crs` folder and extra manifests in Git.
+
+* The `SiteConfig` CR allows multiple extra-manifest paths.
+When file names overlap in multiple directory paths, the last file found in the directory order list takes precedence.
```

Comment:
Meaning is unclear here.
Would something like this be better?
In cases where directories contain files with the same name, the file in the final directory takes precedence.